### PR TITLE
Removes reinforced walls from under grilles on Meta and Delta

### PR DESCRIPTION
**Pull request by dragomagol**
## About The Pull Request
See title.

## Why It's Good For The Game
Wallening QoL; as it is now, reinforced walls look wonky when placed underneath grilles. This is really only a problem on a small part of Meta and a large part of Delta. On Tramstation there are girders under grilles by the AI sat, but those don't cause visual clash like the rwalls do. 

## Changelog
:cl:
qol: removed reinforced walls under grilles in space on Meta and Delta
/:cl:

### CODE REVIEW

**Comment by ShizCalev** [comment]
IIRC those were present to add a little protection during meteor storms.

**Comment by AMonkeyThatCodes** [comment]
then they shouldn't be on the wall.

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 1]
```diff
@@ -102689,6 +102736,13 @@
 /obj/structure/cable,
 /turf/open/floor/iron,
 /area/commons/storage/tools)
+"vxe" = (
+/obj/structure/window/reinforced{
+	dir = 1;
+	layer = 2.9
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:102742]: is this layer intentional?

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 2]
```diff
@@ -126868,24 +126864,24 @@ aox
 dqT
 lMJ
 lMJ
-snH
-snH
-snH
+vrW
```
Comment (ShizCalev) [inline _maps/map_files/MetaStation/MetaStation.dmm:126867]: there's no lattice beneath this grill.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 3]
```diff
@@ -170346,20 +170412,20 @@ aZL
...
-ajr
-ajr
-ajr
-aad
+qYo
+efQ
+efQ
+efQ
+qYo
+efQ
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm:170420]: image
all these too.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: _maps/map_files/Deltastation/DeltaStation2.dmm (hunk 1 above)
Comment (dragomagol) [reply _maps/map_files/Deltastation/DeltaStation2.dmm:102742]: Not sure why that's there but I can remove the layer.

File: _maps/map_files/Deltastation/DeltaStation2.dmm (hunk 3 above)
Comment (dragomagol) [reply _maps/map_files/Deltastation/DeltaStation2.dmm:170420]: Thanks for the catch, I'll get right on it 👍

**Comment by dragomagol** [comment]
Resolved! I've replaced some reinforced windows as well that were varedited 1 pixel.

**Comment by EOBGames** [comment]
@ShizCalev have your concerns with this PR been addressed adequately?

**Comment by LemonInTheDark** [comment]
Frankly I think the issues behind this are more codeside with wallening, though I'm not sure what can be done about them. Frills lay on top of the grills, so things look like shit.

If you're doing this to the gas tanks it's gonna cause issues too mind

**Comment by AMonkeyThatCodes** [comment]
It's not a code issue with the wallening. Grilles aren't supposed to be ontop of walls because a wall is a block and the grille is a block, they aren't supposed to physically be inside of each other lol

**Comment by Cyberboss** [comment]
> 
> 
> > If you're doing this to the gas tanks it's gonna cause issues too mind
> 
> That's true. Would it cause troubles to remove the grilles on the gas tanks?

Don't think so

**Comment by EOBGames** [comment]
alright well, shiz hasn't responded in a bit, if this turns out to cause too many issues we can evaluate how to move forward then

**Pull request merged**